frontendExternalSystem/config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```

Comment:
Make it a symbolic link to ../../../config/checkstyle/checkstyle_allow_java_collections.xml